auditwheel/policy/policy.json

```diff
@@ -40,6 +40,32 @@
          "GCC": ["3.0", "3.3", "3.3.1", "3.4", "3.4.2", "3.4.4", "4.0.0",
                  "4.2.0", "4.3.0"]
      },
+     "lib_whitelist": [
+         "libgcc_s.so.1",
+         "libstdc++.so.6",
+         "libm.so.6", "libdl.so.2", "librt.so.1",
+         "libc.so.6", "libnsl.so.1", "libutil.so.1", "libpthread.so.0",
+         "libX11.so.6", "libXext.so.6", "libXrender.so.1", "libICE.so.6",
+         "libSM.so.6", "libGL.so.1", "libgobject-2.0.so.0",
+         "libgthread-2.0.so.0", "libglib-2.0.so.0", "libresolv.so.2"
+     ]},
+    {"name": "manylinux2014",
+     "priority": 80,
+     "symbol_versions": {
+         "GLIBC": ["2.0", "2.1", "2.1.1", "2.1.2", "2.1.3", "2.2", "2.2.1",
+                   "2.2.2", "2.2.3", "2.2.4", "2.2.5", "2.2.6", "2.3", "2.3.2",
+                   "2.3.3", "2.3.4", "2.4", "2.5", "2.6", "2.7", "2.8", "2.9",
+                   "2.10", "2.11", "2.12", "2.13", "2.14", "2.15", "2.16",
+                   "2.17"],
+         "CXXABI": ["1.3", "1.3.1", "1.3.2", "1.3.3", "1.3.4", "1.3.5",
+                    "1.3.6", "1.3.7", "TM_1"],
```

Comment:
What's the `TM_1` version? Seems odd to be a symbol version. I would take it out unless there is a good reason to keep it.

Reply:
as mentioned in description, all symbols version come from `scripts/calculate_symbol_versions.py`. I've not checked which symbol gets versioned with it.
This version is also present on all distros I checked for compatibility: https://github.com/pypa/manylinux/pull/304#issuecomment-498042781

Reply:
Did you need to modify `calculate_symbol_versions.py` to get it to output this version? Looks like it expects all versions to be integers separated by periods:

https://github.com/pypa/auditwheel/blob/3191806fa535c118d4b521ff701e522e9fb639e4/scripts/calculate_symbol_versions.py#L44

Also note that this script is already doing some filtering of the versions:

https://github.com/pypa/auditwheel/blob/3191806fa535c118d4b521ff701e522e9fb639e4/scripts/calculate_symbol_versions.py#L67

So maybe it does make sense to filter `TM_1` here. (I have no idea what it is.)

Reply:
> Did you need to modify calculate_symbol_versions.py to get it to output this version?

Yes, first commit of this PR

> Also note that this script is already doing some filtering of the versions

It makes sense to filter `PRIVATE` symbols which I guess are used between glibc libraries for internal usage. `TM_1` provides __cxa_tm_cleanup function which is a support function used by the compiler given what I understand from https://gcc.gnu.org/onlinedocs/gcc-7.1.0/libitm/The-libitm-ABI.html. I don't think it makes sense to filter it.

Reply:
> Yes, first commit of this PR

Apologies, I missed this.

Reply:
If we're going to allow the `TM_1` symbol version, presumably the PEP should be updated to mention that?

Reply:
Ping @mayeut. I think this is the last comment which needs addressing. We can then land this PR.

Reply:
Right, I will post a message to libstdc++ mailing list over the weekend to get their input on this symbol version.

Reply:
https://gcc.gnu.org/ml/libstdc++/2019-10/msg00023.html

Reply:
The reply indicates that it can be relied upon, although there's incredulity that a Python extension would ever use it. We're not using it, though, just allowing libraries that may use it. 

So unless there's any reason auditwheel should block libraries from using transactional memory extensions, we can leave it in here and add it to the PEP.

Reply:
Opened a [PEP Pull Request](https://github.com/python/peps/pull/1201)